README.md

```diff
@@ -2,6 +2,8 @@
 
 This model is used to predict the risk of West Nile Virus being present two consecutive weeks in a row. 
 
+Most of the work in this project is data wrangling to get the data into a format where each row represents the results for a week / location collection, and the interval between the dates is regular (i.e. a week apart). Once this is accomplished, a model is fit that is very similar to a generalized linear model to predict the likelihood that WNV will be present if it was present last week. This allows our teams to go out and spray for mosquitos a week earlier than they would have otherwise.
```

Comment:
> I would rather say "This project forecasts the risk of", but I don't really care that much.

Yeah, I think that's right--I was sloppy. Prediction and forecast isn't the same. Borrowing heavily from Nate Silver's book, a prediction is a definitive statement about when something will happen (there will be a positive result tomorrow at trap 123) versus a forecast (there is a 60% chance there will be a positive result tomorrow).

>  Do you think "any mosquito(s)" would be too awkward?

I think that's fine, but I'd ask @thorsean. 

> What do you think about this?

Looks good.

> Do you think we should cite the book here?

I don't think that's necessary. MLM/HML was developed over a number of years.